clients/src/main/java/org/apache/kafka/clients/admin/TopicCollection.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.clients.admin;
+
+import org.apache.kafka.common.Uuid;
+
+import java.util.Collection;
+
+public class TopicCollection {
```

Comment:
Did you make this name generic so that it could handle other use cases?

Reply:
Ideally this would be in a package that can be used by other clients. Is there anything specific to AdminClient here?

Reply:
I was just thinking about the delete/describe use cases. I can move it, but which would be best? I see clients.admin, clients.consumer, clients.producer. Or just clients itself?

Reply:
What about the common package?

Reply:
Ok that would work